pkg/controller/restapi/operation/messaging/models.go

```diff
@@ -0,0 +1,84 @@
+/*
+ *
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * /
+ *
+ */
+
+package messaging
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/controller/command/messaging"
+)
+
+// RegisterMessageServiceRequest model
+//
+// This is used for operation to register a message service to message handler
+//
+// swagger:parameters registerMsgSvc
+type RegisterMessageServiceRequest struct {
+	// Params for registering message service
+	//
+	// in: body
+	Params *messaging.RegisterMsgSvcArgs
+}
+
+// UnregisterMessageServiceRequest model
+//
+// This is used for operation to unregister a message service from message handler
+//
+// swagger:parameters unregisterMsgSvc
+type UnregisterMessageServiceRequest struct {
+	// Params for unregistering a message service
+	//
+	// in: body
+	Params *messaging.UnregisterMsgSvcArgs
+}
+
+// SendNewMessageRequest model
+//
+// This is used for operation to send new message
+//
+// swagger:parameters sendNewMessage
+type SendNewMessageRequest struct {
+	// Params for sending new message
+	//
+	// in: body
+	Params *messaging.SendNewMessageArgs
+}
+
+// SendReplyMessageRequest model
+//
+// This is used for operation to send reply to message
+//
+// swagger:parameters sendReplyMessage
+type SendReplyMessageRequest struct {
+	// Params for sending message reply
+	//
+	// in: body
+	Params *messaging.SendReplyMessageArgs
+}
+
+// RegisterHTTPMessageServiceRequest model
+//
+// This is used for operation to register a HTTP over DIDComm message service to message handler
+//
+// swagger:parameters registerHttpMsgSvc
+type RegisterHTTPMessageServiceRequest struct {
+	// Params for registering http over did comm message service.
+	//
+	// in: body
+	Params *messaging.RegisterHTTPMsgSvcArgs
+}
+
+// RegisteredServicesResponse model
+//
+// This is used for returning list of registered service names
+//
+// swagger:response registeredServicesResponse
+type RegisteredServicesResponse struct {
```

Comment:
Do these structs actually get used somewhere?

Reply:
These structs wrap actual model objects so that swagger export them in spec. If we are ok to have swagger annotations on command models (inside command pkg), we can remove these. What you think?

Reply:
If these structs aren't used it would be good to unexport them and/or put them in a separate file with a build constraint (assuming that swagger can still generate from them).

Reply:
unexported them.